### PR TITLE
fix: Dynamic bhashini proxy endpoint

### PR DESCRIPTION
**Pull request by Ajay-Maury**

### CODE REVIEW

File: src/app.service.ts [hunk 1]
```diff
@@ -1582,31 +1582,33 @@ export class AppService {
...
   }
 
   public async bhashiniProxy(
-    apiUrl: string,
+    endpoint: string,
     req: FastifyRequest,
   ): Promise<AxiosResponse<any>> {
     try {
-      this.logger.debug('Calling bhashini proxy service')
+      this.logger.debug('Calling bhashini proxy service');
```
Comment (tushar5526) [inline src/app.service.ts:1595]: let's debug log the endpoint ?

**Comment by tushar5526** [comment]
Have you tested this with different endpoints ?

File: src/app.service.ts [hunk 2]
```diff
@@ -1582,31 +1582,33 @@ export class AppService {
...
       const authorizationHeader = req.headers['authorization'];
```
Comment (tushar5526) [inline src/app.service.ts:1596]: I believe you should pass all the headers forward.

File: src/app.service.ts [hunk 3]
```diff
@@ -1582,31 +1582,33 @@ export class AppService {
...
       const contentTypeHeader = req.headers['content-type'];
-
+      
       const headers = {
```
Comment (tushar5526) [inline src/app.service.ts:1599]: Pass all the headers from the actual request ?

**Comment by Ajay-Maury** [comment]
> Have you tested this with different endpoints ?

Yes

**Review by tushar5526 — APPROVED** [review]

**Pull request merged**